packages/clay-form/stories/index.tsx

```diff
@@ -76,6 +76,16 @@ const moveBoxesOptions = [
 
 storiesOf('Components|ClayInputMoveBoxes', module).add('default', () => (
 	<ClayInputMoveBoxes
+		// Maybe we want to do something like this for left/right boxes
+		//
+		// left={{
+		// 	items: // []
+		// 	label: //
+		// 	onChange: //
+		// 	selected: //
+		// 	onSelectChange: //
+		// }}
```

Comment:
So this pattern is pretty, but I don't get what the value of `onChange` would be. Would the purpose, and the expectation, be that the dev who uses this component will be able to provide their own function?

Reply:
@kresimir-coko  `onChange` would be the callback for whenever the items in the box are changed. It would be the same callback for if we just used the `SelectBox` component.

```jsx
<ClaySelectBox
	items: // items rendered in list
	onChange: // callback for when items are changed. Like when they are reordered(up/down).
	selected: // value for which items are selected
	onSelectChange: // callback for when the selected items change.
/>
```

Essentially we need a way to know when the items in the list are changed. They could change by being removed, added, or re-ordered.

Added/Removed: when an item is moved between boxes.
Reordered: when an item is moved up/down in the list.